### DISCUSSION TITLE
MIMIC-IV Concepts - No concept index provided

### DISCUSSION THREAD

**Discussion by mdbotto** [MIT-LCP/mimic-code#1510] [MIMIC-IV]
Hello! Would it be possible to provide the Concept Index on the Readme.md file inside the concepts folder please?

**Comment by alistairewj** [comment]
Ah, work in progress for the MIMIC-IV readme. However, many of the concepts overlap with MIMIC-III, so for now you could look there: https://github.com/MIT-LCP/mimic-code/tree/main/mimic-iii/concepts
